src/MyClass.java

```diff
@@ -0,0 +1,8 @@
+public class MyClass {
+    public static void main(String[] args) {
+        int a = 2;
+        int b = 3;
+        int c = a + b;
```

Comment:
it's method implementation so it's no needed to place it in the main method. The main method is for calling others methods